sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -41,18 +39,20 @@ public static DirectConnectionConfig getDefaultConfig() {
         return new DirectConnectionConfig();
     }
 
-    //  TODO: (DANOBLE) - To update these docs.
-
     /**
-     * Gets the direct connection timeout
+     * Gets the connection timeout for direct client,
+     * represents timeout for establishing connections with an endpoint
+     *
      * @return direct connection timeout
      */
     public Duration getConnectionTimeout() {
```

Comment:
How about stating how these manifest for netty underlying for advanced users?

Reply:
Sure, will attach respective Netty links here.